.github/workflows/ci.yml

```
@@ -74,28 +74,28 @@
         with:
           path: test/support/plts
           key: test/support/plts-${{ runner.os }}-${{ matrix.elixir }}-${{ matrix.otp }}-${{ hashFiles(format('{0}{1}', github.workspace, '/mix.lock')) }}
-        if: ${{ contains(matrix.elixir, '1.17.2') && contains(matrix.otp, '27.0') }}
+        if: ${{ contains(matrix.elixir, '1.17.3') && contains(matrix.otp, '27.1') }}
       - name: Get dependencies
         run: mix deps.get
       - name: Compile dependencies
         run: MIX_ENV=test mix deps.compile
```

actionlint

shellcheck reported issue in this script: SC2209:warning:1:1: Use var=$(command) to assign output (or quote to assign string)

(shellcheck)